draft-moriarty-attestationsets.md

```diff
@@ -3,7 +3,7 @@ v: 3
 
 title: Scalable Remote Attestation for Systems, Containers, and Applications
 abbrev: SRASCA
-docname: draft-moriarty-attestationsets-latest
+docname: draft-moriarty-rats-posture-assessment-latest
```

Comment:
@KME, did you explicitly want to rename the this draft via the `docname` change? I saw this in the version you had sent along but I did not know if this was to figure our GitHub issues or because it was a meaningful change, let me know.

Reply:
So I will have to rename the file when porting the changes from [the other repo](https://github.com/KME/draft-moriarty-rats-posture-assessment) back into here to suppress a new transient error besides the one I explain below.

Reply:
Henk requested it and I am okay with the change. I can link the documents in the datatracker when posted.